tslearn/docs/examples/plot_shapelets.py

```diff
@@ -62,8 +74,7 @@
 plt.tight_layout()
 plt.show()
 
-# The loss history is accessible via the `model` attribute that is a keras
-# model
+# The loss history is accessible via the `model` that is a keras model
```

Comment:
`model_` no?

Reply:
Fixed. This was something weird with my git (I never touched this part)